modules/integration-tests/src/main/java/org/apache/fluo/integration/client/FluoClientIT.java

```diff
@@ -83,4 +97,72 @@ public void testFailures() {
     Logger.getLogger(FluoClientImpl.class).setLevel(clientLevel);
     Logger.getLogger(FluoFactory.class).setLevel(factoryLevel);
   }
+
+  @Test(expected = CommitException.class)
+  public void testWriteWithDefaultAuths() throws Throwable {
+    Column labeledColumn = new Column("data", "private_column", "PRIVATE");
+    try (FluoClient client = FluoFactory.newClient(config);
+        Transaction txn = client.newTransaction()) {
+      txn.set("bill", labeledColumn, "a value");
+      txn.commit();
+    }
+  }
+
+  @Test
+  public void testScanTimeAuths() throws Throwable {
+    try (AccumuloClient accumulo = AccumuloUtil.getClient(config)) {
+      accumulo.securityOperations().changeUserAuthorizations(config.getAccumuloUser(),
+          new Authorizations("PRIVATE", "PUBLIC"));
+    }
+
+    Column ssn = new Column("", "ssn", "PRIVATE");
+    Column name = new Column("", "name", "PUBLIC");
+    try (FluoClientImpl client = (FluoClientImpl) FluoFactory.newClient(config)) {
+      client.useAuthorizations(ImmutableList.of("PRIVATE", "PUBLIC"));
+      try (Transaction txn = client.newTransaction()) {
+        txn.set("bill", ssn, "1");
+        txn.set("bill", name, "william");
+        txn.set("bob", ssn, "2");
+        txn.set("bob", name, "robert");
+        txn.commit();
+      }
+
+      System.out.println("***********************************");
+      System.out.println("Default auths");
+      System.out.println("***********************************");
+      try (Snapshot snapshot = client.newSnapshot()) {
+        System.out.println(snapshot.get(Bytes.of("bill")));
```

Comment:
snapshot has multiple get methods, some use scanner and some use batch scanner.  Would be nice to test them all.